### DISCUSSION TITLE
Not able to capture screenshot on computer on which I didn't build the app

### DISCUSSION THREAD

**Discussion by developersaintt** [bencevans/screenshot-desktop#285] [Q&A]
So I have noticed that if I build the app on my computer and then copy the exe to another computer and then run that then the app doesn't capture the screenshot.

what could be the issue?

**Comment by bencevans** [comment]
🤔 what are you using for packaging the application? Electron? What error do you get when running on the second computer?

**Reply by developersaintt** [reply]
I'm using electron builder. I'm not getting any error, its just that the then callback of  the screenshot() is not running
```
console.log("this line is logging")
screenshot().then((img) => {
       console.log("this line is not logging")
        event.sender.send('img-blob', img)
      }).catch(err => {
        console.log(err)
        throw err
      })
```
